include/rapid_pbd/robot_config.h

```diff
@@ -60,6 +62,26 @@ class FetchRobotConfig : public RobotConfig {
   void joints_for_group(const std::string& actuator_group,
                         std::vector<std::string>* joint_names) const;
   int num_arms() const;
+  std::string joint_states_topic() const;
+};
+
+class BaxterRobotConfig : public RobotConfig {
+ public:
+  BaxterRobotConfig();
+  std::string planning_frame() const;
+  std::string planning_group() const;
+  std::string base_link() const;
+  std::string torso_link() const;
+  std::string ee_frame_for_group(const std::string& actuator_group) const;
+  void gripper_joints_for_group(const std::string& actuator_group,
+                                std::vector<std::string>* joint_names) const;
+  void gripper_open_positions(std::vector<double>* positions) const;
+  void gripper_close_positions(std::vector<double>* positions) const;
+  void default_gripper_poses(std::vector<double>* poses) const;
```

Comment:
Remove this, looks like it's unused.

Reply:
Ah sorry, it was used for my condition learning